src/consumer/subscriptionState.js

```diff
@@ -0,0 +1,26 @@
+module.exports = class SubscriptionState {
+  constructor() {
+    this.pausedTopics = new Set()
```

Comment:
fancy

Reply:
Don't want any of those filthy duplicates in here. 💅